### DISCUSSION TITLE
Hiring: Rust Developer (Berlin, Germany / Remote)

### DISCUSSION THREAD

**Discussion by metabuild-dev** [bevyengine/bevy#3011] [General]
Hey everyone, we at Metabuild are currently looking to hire a rust developer to help
us develop architectural design tools based on Bevy.

We are already using the `Bevy ECS` for some of our internal simulation tools and 
are hoping to expand this to graphical tools we are currently developing.

The offered position can either be in our Berlin office or completly remote.

For more information see: [Job Posting](https://www.metabuild.io/en/careers/rust-developer-m-f-d/)

If you are interested please reach out to us at jobs@metabuild.io

## About us

Metabuild is a startup based in Berlin, Germany. We are a team of innovation driven experts bridging the gap between architecture, engineering and software development.

  
Our building simulation and optimization tools offer architects and planners the insights needed to optimize energy efficiency, occupant comfort and cost effectiveness.

**Comment by TheRawMeatball** [comment]
If you have Discord, I'd recommend also posting this on #job-postings in the official bevy discord.

**Reply by metabuild-dev** [reply]
Thanks for the info. 
I will do so :+1:

**Comment by Gavolot** [comment]
Is this still relevant?
